lib/src/authorization_request.dart

```diff
@@ -12,7 +12,8 @@ class AuthorizationRequest extends CommonRequestDetails
       Map<String, String> additionalParameters,
       String issuer,
       String discoveryUrl,
-      List<String> promptValues}) {
+      List<String> promptValues,
+      bool requireHttps}) {
```

Comment:
Can you rename all instances of `requireHttps` on the Dart and Java side to something like `allowInsecureConnections`? You should also be able to provide a default value